VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -159,32 +171,127 @@ final public class AuthenticationController
     
     public func login(email email: String, password: String, completion: AuthenticationCompletion)
     {
-        let request = AuthenticationRequest.postLoginRequest(email: email, password: password, scopes: self.configuration.scopes)
+        let request = AuthenticationRequest.loginRequest(email: email, password: password, scopes: self.configuration.scopes)
```

Comment:
Extreme nit: 

We say `logOut` but `login`, and Tamara prefers "logIn", change to "logIn"?